src/platform/monitoring/DowntimeNotification/actions/index.js

```diff
@@ -54,47 +51,24 @@ export function dismissDowntimeWarning(appTitle) {
   };
 }
 
-export function getScheduledDowntime(downtimeWindow = scheduledDowntimeWindow) {
+export const getGlobalDowntime = () => async dispatch => {
+  const downtime = await getCurrentGlobalDowntime();
+  dispatch({ type: RECEIVE_GLOBAL_DOWNTIME, downtime });
+};
+
+export function getScheduledDowntime() {
   return async dispatch => {
     dispatch({ type: RETRIEVE_SCHEDULED_DOWNTIME });
 
-    // create global downtime data if feature toggle is enabled and if
-    // current time is in downtime window
-    // default to empty array
-    const { downtimeStart, downtimeEnd } = downtimeWindow;
-
-    const globalDowntimeData =
-      (!environment.isLocalhost() &&
-        moment().isAfter(downtimeStart) &&
-        moment().isBefore(downtimeEnd) &&
-        createGlobalMaintenanceWindow({
-          startTime: downtimeStart,
-          endTime: downtimeEnd,
-        })) ||
-      [];
+    let data;
 
     try {
-      await apiRequest('/maintenance_windows/')
-        .then(({ data }) => {
-          if (data.errors?.[0].status === '500') {
-            Promise.reject();
-          }
-          dispatch({
-            type: RECEIVE_SCHEDULED_DOWNTIME,
-            data,
-          });
-        })
-        .catch(() =>
-          dispatch({
-            type: ERROR_SCHEDULE_DOWNTIME,
-            data: globalDowntimeData,
-          }),
-        );
-    } catch (err) {
-      dispatch({
-        type: ERROR_SCHEDULE_DOWNTIME,
-        data: globalDowntimeData,
-      });
+      const response = await apiRequest('/maintenance_windows');
+      data = response.data;
+    } catch (error) {
+      // Probably in a test environment and the route isn't mocked.
+    } finally {
+      dispatch({ type: RECEIVE_SCHEDULED_DOWNTIME, data });
```

Comment:
This is basically a revert to before we tried to incorporate the hard-coded maintenance window into this action creator.